mcc/odin/odin/snapshot_list_test.go

```diff
@@ -2,25 +2,41 @@ package odin_test
 
 import (
 	"testing"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/rds"
 
 	"github.com/poka-yoke/spaceflight/mcc/odin/odin"
 )
 
+const (
+	RFC8601 = "2016-01-02T15:04:05+07:00"
```

Comment:
shouldn't this be a format sting inluding stuff like `%d` or `%y` instead of a exact date string? otherwise what does `time.Parse` do? Take the format of one time and apply it to another?